nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -997,6 +997,19 @@ services.transmission.settings.rpc-bind-address = "0.0.0.0";
       the previous behaviour using <literal>undervolt.useTimer</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>deepin</literal> package set has been removed from
+      nixpkgs. This work was still in progress to package the
+      <link xlink:href="https://www.deepin.org/en/dde/">Deepin Desktop Environment (DDE)</link>,
+      including libraries, tools and applications, but still missing a
+      service to lauch the desktop environment. It has shown to be a
+      difficult task to complete, as discussed in
```

Comment:
It has shown to no longer be a feasible goal due to reasons discussed in-
I think that's the more definitive conclusion to why we removed it.

Reply:
Done.